### ISSUE TITLE
Update Slicer icon on Wikipedia

### ISSUE THREAD

**Issue by lassoan** [Slicer/Slicer#6072]
The new Slicer5 logo has been rolled out in 2021 Fall. Accordingly, the Slicer logo need to be updated at https://en.wikipedia.org/wiki/3DSlicer

**Comment by pieper** [comment]
I tried editing the wikipedia page, but the image upload was disallowed by some automated filter.  I filed an issue so we'll see what happens.

**Comment by pieper** [comment]
Here's [the response](https://commons.wikimedia.org/wiki/Commons_talk:Abuse_filter#c-Jeff_G.-2021-12-07T13%3A36%3A00.000Z-Pieper923-2021-12-07T13%3A28%3A00.000Z) I got from wikipedia.  Looks like some extra steps need to be taken to register the logo with them.

Report by Pieper923
Report by: Pieper923 (talk • contribs • block log • filter log)
Page name: https://en.wikipedia.org/wiki/3D_Slicer (https://en.wikipedia.org/wiki/3D_Slicer links: hist • logs • abuse log • en:https://en.wikipedia.org/wiki/3D_Slicer)
Edit kind: Trying to update the logo with the latest version
Comments: I'm not sure how else I would be able to change the logo. It seems this filter will block anything I would try to do, even though it's a legitimate edit. Pieper923 (talk) 13:28, 7 December 2021 (UTC)
@Pieper923: Hi, and welcome. I am sorry to inform you that you have triggered Special:AbuseFilter/153 by trying to cross-wiki upload a png logo as a new user while leaving the summary intact. Such uploads of png logos are not allowed at all. You indicated it's your own work. Usually when someone uploads a png logo, it's a copyright violation taken from the web. Please upload the full-size original of it per COM:HR, including any metadata, but it may be judged too complex to be under TOO in the country of origin, so you may need to license it on your official website or social media or send permission via VRT. Also, any png image will look fuzzy when scaled down (due to design decisions discussed in phab:T192744) or jaggy when scaled up, so you may want to upload an svg or jpg version, too. If you can't get a compliant license, the logo may still be uploaded to English Wikipedia in compliance with en:WP:F because we don't allow Fair Use here. If you change the summary or use our Upload Wizard instead, you should be able to avoid that filter.   — Jeff G. ツ please ping or talk to me 13:36, 7 December 2021 (UTC)

**Comment by luzpaz** [comment]
Is this still relevant ?

**Comment by lassoan** [comment]
Yes, it is still relevant. The wikipedia entry still shows the old Slicer logo.
